analysis2_manuscript.tex

```diff
@@ -741,6 +743,8 @@ \section*{Detection of Selective Sweeps}
     In the YRI sample, \diploshic outperforms the other two methods,
     with an AUC of 0.979, followed by using outliers of $\pi$ with an AUC of 0.957,
     and then \sweepfinder with an AUC of 0.939.
+    Note, however, that despite the apparent good performance of the $\pi$ statistic,
+    it showed extreme heterogeneity across the genome. % MFR: I think this is an important point. Basically there is a Simpson's paradox here.
```

Comment:
@mufernando -- you aren't happy with how we say this a few lines down?

Reply:
oops, I missed this. I removed the new sentence.